db/seeds.rb

```diff
@@ -165,50 +159,159 @@
 User.create!(
   email: 'tupac@gmail.com',
   password: 'password',
-  nickname: 'QO',
+  nickname: 'Grubby',
   show_game: csgo,
   game: csgo,
   rank: Rank.find_by(id: 42),
   position_name_id: csgo.position_names.first.id)
 
 puts 'Users created'
 
-
 puts 'Creating Teams'
 
-Team.create!(
-  name: 'Team Secret',
+team_logo = URI.open('https://res.cloudinary.com/dm4nbcc5v/image/upload/v1584528718/logos_for_seeds/chaos_pgwdak.png')
+team1 = Team.new(
+  name: 'Chaos eSports',
   game_id: 1,
-  user_id: 1)
+  user_id: 2)
+team1.photo.attach(io: team_logo, filename: 'chaos.png', content_type: 'image/png')
```

Comment:
is this storing locally? Or does the .attach automatically upload to cloudinary?

Reply:
This is automatically uploading to cloudinary.